assemblies/lib/pom.xml

```diff
@@ -236,11 +236,17 @@
       <version>${platform.version}</version>
       <exclusions>
         <exclusion>
-          <groupId>*</groupId>
+          <groupId>*</groupId> <!-- not all dependencies should be excluded, snakeyaml is needed forr KarafBoot.java -->
```

Comment:
The library snakeyaml-2.2.jar ( "org.yaml:snakeyaml:2.2") replaces the previous interations of safeyaml-1.34.1.jar ( "org.hitachivantara:safeyaml" and "com.konloch:safeyaml") and snakeyaml-1.33.jar (""org.yaml:snakeyaml:1.33")

Previous maven configurations had the dependency for `snakeyaml.jar` coming in from `kettle-engine.jar` which was incorrect, since no code uses Yaml library. It didn't matter since both `kettle-engine.jar` and `pentaho-platform-extensions.jar` were always in the same "library" folder.  At the end of the day, `snakeyaml.jar` is a runtime dependency for `pentaho-platform-extensions.jar` and they need to be in the same folder and/or on the same class path.

For pentaho reporting the "library" folders:
- PDI-CE (before)
  - data-integration/lib/pentaho-platform-extensions.jar
  - **data-integration/lib/safeyaml.jar**
- PRD-CE (after)
  - data-integration/lib/pentaho-platform-extensions.jar
  - **data-integration/lib/snakeyaml.jar**

PDI-CE feeds into PDI-EE so  no need to duplicate this configuration in PDI-EE pom.xmls

Any attempt to bring all the runtime jars for `pentaho-platform-extensions.jar` resulted in bringing in 100+ jars. Re-configuring multiple maven poms is out of scope for this issue.

For more detailed explanation and comment: https://hv-eng.atlassian.net/browse/PPP-5163?focusedCommentId=2073002

Reply:
non-blocker, typo in "forr KarafBoot.java"